exporters/prometheus/exporter.go

```diff
@@ -148,6 +148,9 @@ func (c *collector) Collect(ch chan<- prometheus.Metric) {
 	metrics := metricdata.ResourceMetrics{}
 	err := c.reader.Collect(context.TODO(), &metrics)
 	if err != nil {
+		if err == metric.ErrReaderShutdown {
```

Comment:
Can we use `errors.Is` instead of direct comparison? That should future proof any wrapping of that error.

Reply:
done